simapp/simd/cmd/root.go

```diff
@@ -75,6 +79,43 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// initConfig helps to override default appConfig template and settings.
+// return; if no custom configuration is required for the application.
+func initConfig() (string, interface{}) {
```

Comment:
could you add a few short sentences in the docs about this please.

Reply:
oh sorry, meant the docs/ directory. I see you added in the doc.go.

Reply:
I didn't find a relavant place to add it actually. There are no docs explaining server configs. May be I will add a new file or a new section under `node` docs.

Reply:
updated `rootCmd` section docs